mods/cnc/maps/gdi01/map.yaml

```diff
@@ -424,9 +424,6 @@ Rules:
 		-SpawnMPUnits:
 		-MPStartLocations:
 		-CrateSpawner:
-		PlayMusicOnMapLoad:
```

Comment:
99% of the maps at http://resource.openra.net/ don't use it and the rest who does knows enough about the engine to fix it easily so I don't see that as a blocker. This is already a very long-running pull request with many additional fixes and requests made.